unity/ProjectSettings/ProjectSettings.asset

```diff
@@ -13,15 +13,15 @@ PlayerSettings:
   useOnDemandResources: 0
   accelerometerFrequency: 60
   companyName: Allen Institute for Artificial Intelligence
-  productName: AI2-Thor
+  productName: AI2-THOR
```

Comment:
Can you revert this change for now? It will affect the executable name generated by the build which we assume to be 'AI2-Thor'.

Reply:
Hmm... Perhaps, although this change was intentional. AI2-Thor is an incorrect spelling that doesn't look that good (should be either AI2-THOR or ai2thor).

That said, it doesn't affect the demo in any way, so I'm fine reverting it for now.

Reply:
I agree that the spelling should be corrected.  I think it got set very early in the life of the project but never changed.  Altering the spelling complicates things a bit since we assume the binary name to be 'AI2-Thor' currently.  This really only affects OSXIntel64 and can update how we look for the executable path to make it handle name changes properly.  Can you create an issue and assign it to me to support this?